Source/Backlang.Codeanalysis/Parsing/Parser.Expressions.cs

```diff
@@ -125,8 +125,14 @@ private LNode ParseNumber()
             }
 
             Iterator.NextToken();
+        } else if (result.Value is double)
+        {
+            result = SyntaxTree.Unary(Symbols.Float64, result);
```

Comment:
Float64 is not an unary operator. Float64 is a simple LNode.Call. Same For Int32